README.md

```diff
@@ -246,6 +270,24 @@ module.exports = {
 }
 ```
 
+When using `.graphqlconfig`, you might end up not needing to specify any additional options for a rule. Note that you still _have_ to provide an empty object (`{}`) to the rule:
```

Comment:
this API is a little strange to suggest. [can we default the optionGroup to an empty object?](https://github.com/apollographql/eslint-plugin-graphql/blob/master/src/index.js#L246-L254) that should (hopefully) resolve some of this wonkiness

Reply:
I'll see if I can come up with something :)

Reply:
I pushed a fix now. (Note that this would also be fixed by #87).